libcontainer/cgroups/systemd/v2.go

```diff
@@ -292,6 +292,24 @@ func (m *unifiedManager) Apply(pid int) error {
 	if err := fs2.CreateCgroupPath(m.path, m.cgroups); err != nil {
 		return err
 	}
+
+	// When systemd creates a unit owned by a user other than the
+	// user running the service manager (usually root), it chowns
+	// the cgroup directory and the three files mentioned below.
+	// This enables the service to create new scopes/slices and set
+	// the cgroup for its child processes/threads.
+	//
+	// We follow systemd's lead and chown only those same files.
+	filesToChown := []string{".", "cgroup.procs", "cgroup.subtree_control", "cgroup.threads"}
```

Comment:
I remembered there's a cgroup file which lists these three, but can't find it now. Maybe I remember wrong :)

Reply:
@kolyshkin I am not aware of it.

Reply:
This is also documented in the kernel docs: 
https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#model-of-delegation

(no need to add a link I guess, just noticing)